cmd/cluster/context.go

```diff
@@ -201,46 +201,40 @@ func (o *contextOptions) run() error {
 
 func (o *contextOptions) printLongOutput(data *contextData) {
 
+	clusterHeader := fmt.Sprintf("%s -- %s", data.ClusterName, data.ClusterID)
+	fmt.Println(clusterHeader)
+	fmt.Println(strings.Repeat("=", len(clusterHeader)))
+
 	printClusterInfo(o.clusterID)
 
 	printSupportStatus(data.LimitedSupportReasons)
-
-	// Print the Servicelogs for this cluster
+	fmt.Println()
+	printJIRASupportExceptions(data.SupportExceptions)
+	fmt.Println()
 	printServiceLogs(data.ServiceLogs, o.verbose, o.days)
-
-	printJIRAOHSS(data.JiraIssues)
-
+	fmt.Println()
+	printJiraIssues(data.JiraIssues)
+	fmt.Println()
 	printCurrentPDAlerts(data.PdAlerts, data.pdServiceID)
+	fmt.Println()
 
 	if o.full {
-		_ = printHistoricalPDAlertSummary(data.HistoricalAlerts, data.pdServiceID, o.days)
+		printHistoricalPDAlertSummary(data.HistoricalAlerts, data.pdServiceID, o.days)
```

Comment:
Is there a reason we didn't handle the error from this function in the past? If not, I think we should start. If so, I think not intentionally assigning the output to the blank identifier might trigger gosec's compliance check?

Reply:
That's fair - I don't know why it was being ignored before.  The only possible error to be returned was a table.Flush() error, but we can still catch and handle that.

Reply:
Alright, since the error return for each of the print* functions was just output from `table.Flush()`, I've changed all of them to be consistent with one another.  They will:

* no longer return errors, but
* if an error is encountered, print it to stderr, so the SRE will see the action-specific error printed under the action header, or, if sending stderr elsewhere, see the errors collected, with the action name + the error it encountered.

This should standardize how the errors are handled, raise them to the SREs with appropriate context, and remove the need to handle any return from the various printer functions.

Reply:
Agreed, it makes the most sense to complete the context evaluation even if an error is encountered rather than bail early and potentially miss out on important information :+1: